rc1

ocefpaf opened the pull request:
Ping @jorisvandenbossche

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

jorisvandenbossche commented:
@ocefpaf thanks for keeping those up to date.

Don't know if it was on purpose or not, but note that the rc has been added to the main channel and not dev channel again.

ocefpaf commented:
> Don't know if it was on purpose or not, but note that the rc has been added to the main channel and not dev channel again.

Not on purpose. I was sure I fixed that in #103 but clearly something else is amiss :-/

Thanks for the heads up. I fixed that manually and the dev releases are back in the dev channel.
